### PR TITLE
feat: Enable testing with Nix

### PR DESCRIPTION
**Pull request by l0b0**

### CODE REVIEW

File: Makefile.PL [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Note: this file was auto-generated by Module::Build::Compat version 0.4231
```
Comment (MitchellPaff) [inline Makefile.PL:1]: Is this file intentional?

Comment (l0b0) [reply Makefile.PL]: Nope, fixed.

**Review by MitchellPaff — APPROVED** [review]
